### PR TITLE
remove authors from fxns

### PR DESCRIPTION
**Pull request by 0xMuluh**

### CODE REVIEW

File: R/addDissimilarity.R [hunk 1]
```diff
@@ -90,15 +90,6 @@
...
 #' 
-#' @author
-#' For overlap implementation: 
-#' Leo Lahti and Tuomas Borman. Contact: \url{microbiome.github.io}
-#' 
-#' For JSD implementation:
-#' Susan Holmes \email{susan@@stat.stanford.edu}.
-#' Adapted for phyloseq by Paul J. McMurdie.
-#' Adapted for mia by Felix G.M. Ernst
-#'
```
Comment (antagomir) [inline R/addDissimilarity.R]: Just for acknowledgments reasons, move at least the JSD information in @details

File: R/importQIIME2.R [hunk 2]
```diff
@@ -69,7 +69,6 @@
 #' \code{\link[=importMothur]{importMothur}}
 #'
 #' @export
-#' @author Yang Cao
```
Comment (antagomir) [inline R/importQIIME2.R:72]: Let's keep this one for acknowledgment purposes as it is specific case.

File: R/addDissimilarity.R (hunk 1 above)
Comment (0xMuluh) [reply R/addDissimilarity.R]: ACK.

File: R/importQIIME2.R (hunk 2 above)
Comment (0xMuluh) [reply R/importQIIME2.R:72]: ACK.

**Comment by 0xMuluh** [comment]
Ready!

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Closes #628

**Pull request merged**